--- upstream/crates/revmc-codegen/src/tests/macros.rs
+++ evm2/crates/jit/codegen/src/tests/macros.rs
@@ -6,8 +6,8 @@
             #[allow(unused_imports)]
             use similar_asserts::assert_eq;

-            fn run_llvm(compiler: &mut EvmCompiler<revmc_llvm::EvmLlvmBackend>) {
-                if std::env::var_os("REVMC_TEST_DUMP").is_some() {
+            fn run_llvm(compiler: &mut EvmCompiler<evm2_jit_llvm::EvmLlvmBackend>) {
+                if std::env::var_os("EVM2_JIT_TEST_DUMP").is_some() {
                     crate::tests::set_test_dump(compiler, module_path!());
                 }
                 $run(compiler);
@@ -15,13 +15,16 @@

             #[test]
             fn unopt() {
-                crate::tests::with_jit_compiler(revmc_backend::OptimizationLevel::None, run_llvm);
+                crate::tests::with_jit_compiler(
+                    evm2_jit_backend::OptimizationLevel::None,
+                    run_llvm,
+                );
             }

             #[test]
             fn opt() {
                 crate::tests::with_jit_compiler(
-                    revmc_backend::OptimizationLevel::default(),
+                    evm2_jit_backend::OptimizationLevel::default(),
                     run_llvm,
                 );
             }
