--- upstream/crates/revmc-llvm/Cargo.toml
+++ evm2/crates/jit/llvm/Cargo.toml
@@ -1,13 +1,13 @@
 [package]
-name = "revmc-llvm"
+name = "evm2-jit-llvm"
 description = "EVM bytecode compiler LLVM backend"
-homepage = "https://github.com/danipopes/revmc/tree/main/crates/revmc-llvm"

 version.workspace = true
 authors.workspace = true
 edition.workspace = true
 rust-version.workspace = true
 license.workspace = true
+homepage.workspace = true
 categories.workspace = true
 keywords.workspace = true
 repository.workspace = true
@@ -17,7 +17,7 @@
 workspace = true

 [dependencies]
-revmc-backend.workspace = true
+evm2-jit-backend.workspace = true

 inkwell = { version = "0.9.0", features = ["llvm22-1"] }
 alloy-primitives = { workspace = true, features = ["std", "map-fxhash"] }
@@ -26,7 +26,3 @@

 [build-dependencies]
 cc = "1"
-
-[features]
-prefer-static = ["inkwell/llvm22-1-prefer-static"]
-prefer-dynamic = ["inkwell/llvm22-1-prefer-dynamic"]
