--- upstream/crates/revmc-runtime/src/lib.rs
+++ evm2/crates/jit/runtime/src/lib.rs
@@ -7,17 +7,19 @@
 extern crate tracing;
 
 pub use ::eyre;
-pub use revmc_codegen::*;
+pub use evm2_jit_codegen::*;
 
-use revmc_backend::OptimizationLevel;
-use revmc_context::EvmCompilerFn;
+use evm2_jit_backend::OptimizationLevel;
+use evm2_jit_context::EvmCompilerFn;
 #[cfg(feature = "llvm")]
-use revmc_llvm as llvm;
+use evm2_jit_llvm as llvm;
 #[cfg(feature = "llvm")]
-use revmc_llvm::EvmLlvmBackend;
+use evm2_jit_llvm::EvmLlvmBackend;
 
 pub mod runtime;
 
+pub mod evm2_evm;
+
 pub mod revm_evm;
 
 #[cfg(feature = "alloy-evm")]
