--- upstream/crates/revmc-cli/Cargo.toml
+++ evm2/crates/jit/cli/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
-name = "revmc-cli"
+name = "evm2-jit-cli"
 description = "EVM bytecode compiler CLI"
-homepage = "https://github.com/danipopes/revmc/tree/main/crates/revmc-cli"
+homepage = "https://github.com/danipopes/evm2-jit/tree/main/crates/evm2-jit-cli"
 publish = false
 
 version.workspace = true
@@ -21,8 +21,8 @@
 workspace = true
 
 [dependencies]
-revmc.workspace = true
-revmc-statetest = { path = "../revmc-statetest" }
+evm2-jit.workspace = true
+evm2-jit-statetest = { path = "../statetest" }
 
 revm-bytecode.workspace = true
 revm-context = { workspace = true, features = ["std"] }
@@ -47,22 +47,22 @@
 
 [dev-dependencies]
 criterion.workspace = true
-revmc-cli-tests = { path = "../revmc-cli-tests/" }
+evm2-jit-cli-tests = { path = "../cli-tests/" }
 hex = "0.4"
 
 [build-dependencies]
-revmc-build.workspace = true
+evm2-jit-build.workspace = true
 
 [features]
-default = ["llvm-prefer-dynamic", "revmc/asm-keccak"]
-llvm = ["revmc/llvm"]
-llvm-prefer-static = ["llvm", "revmc/llvm-prefer-static"]
-llvm-prefer-dynamic = ["llvm", "revmc/llvm-prefer-dynamic"]
+default = ["llvm-prefer-dynamic", "evm2-jit/asm-keccak"]
+llvm = ["evm2-jit/llvm"]
+llvm-prefer-static = ["llvm", "evm2-jit/llvm-prefer-static"]
+llvm-prefer-dynamic = ["llvm", "evm2-jit/llvm-prefer-dynamic"]
 
 tracy = ["dep:tracing-tracy"]
 
 [[bin]]
-name = "revmc"
+name = "evm2-jit"
 path = "src/main.rs"
 required-features = ["llvm"]
 doc = false
