--- upstream/crates/revmc-build/Cargo.toml
+++ evm2/crates/jit/build/Cargo.toml
@@ -1,13 +1,13 @@
 [package]
-name = "revmc-build"
+name = "evm2-jit-build"
 description = "EVM bytecode compiler build script support"
-homepage = "https://github.com/danipopes/revmc/tree/main/crates/revmc-build"

 version.workspace = true
 authors.workspace = true
 edition.workspace = true
 rust-version.workspace = true
 license.workspace = true
+homepage.workspace = true
 categories.workspace = true
 keywords.workspace = true
 repository.workspace = true
