--- upstream/crates/revmc-context/Cargo.toml
+++ evm2/crates/jit/context/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
-name = "revmc-context"
+name = "evm2-jit-context"
 description = "EVM bytecode compiler runtime context"
-homepage = "https://github.com/danipopes/revmc/tree/main/crates/revmc-context"
+homepage = "https://github.com/danipopes/evm2-jit/tree/main/crates/evm2-jit-context"
 
 version.workspace = true
 authors.workspace = true
@@ -17,6 +17,8 @@
 workspace = true
 
 [dependencies]
+alloy-primitives.workspace = true
+evm2 = { workspace = true, optional = true }
 revm-interpreter.workspace = true
 revm-primitives.workspace = true
 ruint.workspace = true
@@ -30,6 +32,8 @@
 [features]
 default = ["std"]
 std = [
+    "alloy-primitives/std",
+    "evm2?/std",
     "revm-interpreter/std",
     "revm-primitives/std",
     "revm-context?/std",
