--- upstream/crates/revmc-builtins/src/macros.rs
+++ evm2/crates/jit/builtins/src/macros.rs
@@ -37,16 +37,6 @@
 }
 
 #[collapse_debuginfo(yes)]
-macro_rules! pop {
-    ($sp:expr; $($x:ident),* $(,)?) => {
-        $(
-            $sp = $sp.sub(1);
-            let $x = &mut *$sp;
-        )*
-    };
-}
-
-#[collapse_debuginfo(yes)]
 macro_rules! try_into_usize {
     ($x:expr) => {
         match $x.to_u256().as_limbs() {
