--- upstream/examples/runner/build.rs
+++ evm2/crates/jit/examples/runner/build.rs
@@ -1,4 +1,4 @@
-use revmc::{EvmCompiler, Result, SpecId, primitives::hex};
+use evm2_jit::{EvmCompiler, Result, SpecId, primitives::hex};
 use std::path::PathBuf;
 
 include!("./src/common.rs");
@@ -6,7 +6,7 @@
 fn main() -> Result<()> {
     // Emit the configuration to run compiled bytecodes.
     // This not used if we are only using statically linked bytecodes.
-    revmc_build::emit();
+    evm2_jit_build::emit();
 
     // Compile and statically link a bytecode.
     let name = "fibonacci";
