--- upstream/examples/runner/src/lib.rs
+++ evm2/crates/jit/examples/runner/src/lib.rs
@@ -6,18 +6,18 @@
 
 // This dependency is needed to define the necessary symbols used by the compiled bytecodes,
 // but we don't use it directly, so silence the unused crate dependency warning.
-use revmc_builtins as _;
+use evm2_jit_builtins as _;
 
 use revm_context::{BlockEnv, CfgEnv, Context, Journal, TxEnv};
 use revm_database_interface::Database;
 use revm_handler::{MainBuilder, MainnetEvm};
 use revm_primitives::{B256, hardfork::SpecId, hex, map::B256Map};
-use revmc_context::{JitEvm, RawEvmCompilerFn};
+use evm2_jit_context::{JitEvm, RawEvmCompilerFn};
 
 include!("./common.rs");
 
 // The bytecode we statically linked.
-revmc_context::extern_revmc! {
+evm2_jit_context::extern_revmc! {
     fn fibonacci;
 }
 
