--- upstream/fuzz/Cargo.toml
+++ evm2/crates/jit/fuzz/Cargo.toml
@@ -1,5 +1,5 @@
 [package]
-name = "revmc-fuzz"
+name = "evm2-jit-fuzz"
 version = "0.0.0"
 publish = false
 edition = "2021"
@@ -9,11 +9,11 @@
 
 [dependencies]
 libfuzzer-sys = "0.4"
-revmc = { workspace = true, features = ["llvm-prefer-dynamic", "__fuzzing"] }
+evm2-jit = { workspace = true, features = ["llvm-prefer-dynamic", "__fuzzing"] }
 eyre.workspace = true
 
 [build-dependencies]
-revmc-build.workspace = true
+evm2-jit-build.workspace = true
 
 [[bin]]
 name = "vs_interpreter"
