--- /home/doni/github/paradigmxyz/revm-inspectors/src/tracing/fourbyte.rs	2025-07-10 16:22:55.681658487 +0200
+++ /home/doni/github/danipopes/evm2.1/crates/inspectors/src/tracing/fourbyte.rs	2026-05-15 03:42:31.933960877 +0200
@@ -1,4 +1,4 @@
-//! Fourbyte tracing inspector
+//! Fourbyte tracing inspector.
 //!
 //! Solidity contract functions are addressed using the first four bytes of the Keccak-256 hash of
 //! their signature. Therefore when calling the function of a contract, the caller must send this
@@ -20,52 +20,37 @@
 //! ```
 //!
 //! See also <https://geth.ethereum.org/docs/developers/evm-tracing/built-in-tracers>
+
 use alloc::format;
-use alloy_primitives::{hex, map::HashMap, Selector};
+use alloy_primitives::{Selector, hex, map::HashMap};
 use alloy_rpc_types_trace::geth::FourByteFrame;
-use revm::{
-    context::{ContextTr, LocalContextTr},
-    interpreter::{CallInput, CallInputs, CallOutcome},
-    Inspector,
+use evm2::{
+    EvmTypes, Inspector,
+    interpreter::{Message, MessageResult},
 };
 
 /// Fourbyte tracing inspector that records all function selectors and their calldata sizes.
 #[derive(Clone, Debug, Default)]
 pub struct FourByteInspector {
-    /// The map of SELECTOR to number of occurrences entries
+    /// The map of SELECTOR to number of occurrences entries.
     inner: HashMap<(Selector, usize), u64>,
 }
 
 impl FourByteInspector {
-    /// Returns the map of SELECTOR to number of occurrences entries
+    /// Returns the map of SELECTOR to number of occurrences entries.
     pub const fn inner(&self) -> &HashMap<(Selector, usize), u64> {
         &self.inner
     }
 }
 
-impl<CTX: ContextTr> Inspector<CTX> for FourByteInspector {
-    fn call(&mut self, context: &mut CTX, inputs: &mut CallInputs) -> Option<CallOutcome> {
-        if inputs.input.len() >= 4 {
-            let r;
-            let input_bytes = match &inputs.input {
-                CallInput::SharedBuffer(range) => {
-                    match context.local().shared_memory_buffer_slice(range.clone()) {
-                        Some(slice) => {
-                            r = slice;
-                            &*r
-                        }
-                        None => &[],
-                    }
-                }
-                CallInput::Bytes(bytes) => bytes.as_ref(),
-            };
-
+impl<T: EvmTypes> Inspector<T> for FourByteInspector {
+    fn call(&mut self, message: &mut Message<T>, _host: &mut T::Host) -> Option<MessageResult<T>> {
+        if message.input.len() >= 4 {
             let selector =
-                Selector::try_from(&input_bytes[..4]).expect("input is at least 4 bytes");
-            let calldata_size = input_bytes[4..].len();
+                Selector::try_from(&message.input[..4]).expect("input is at least 4 bytes");
+            let calldata_size = message.input[4..].len();
             *self.inner.entry((selector, calldata_size)).or_default() += 1;
         }
-
         None
     }
 }
