--- /home/doni/github/paradigmxyz/revm-inspectors/src/tracing/arena.rs	2025-10-29 11:07:06.983570679 +0100
+++ /home/doni/github/danipopes/evm2.1/crates/inspectors/src/tracing/arena.rs	2026-05-15 03:39:43.912562258 +0200
@@ -29,7 +29,7 @@
 
     /// Returns a mutable reference to the nodes in the arena.
     #[inline]
-    pub fn nodes_mut(&mut self) -> &mut Vec<CallTraceNode> {
+    pub const fn nodes_mut(&mut self) -> &mut Vec<CallTraceNode> {
         &mut self.arena
     }
 
