--- upstream/src/tracing/js/builtins.rs
+++ evm2/src/tracing/js/builtins.rs
@@ -1,16 +1,16 @@
 //! Builtin functions
 
 use alloc::{borrow::Cow, format, string::ToString, vec::Vec};
-use alloy_primitives::{hex, map::HashSet, Address, FixedBytes, B256, U256};
+use alloy_primitives::{Address, B256, FixedBytes, U256, hex, map::HashSet};
 use boa_engine::{
+    Context, JsArgs, JsBigInt, JsError, JsNativeError, JsResult, JsString, JsValue, NativeFunction,
+    Source,
     builtins::{array_buffer::ArrayBuffer, typed_array::TypedArray},
     js_string,
     object::builtins::{JsArray, JsArrayBuffer, JsTypedArray, JsUint8Array},
     property::Attribute,
-    Context, JsArgs, JsBigInt, JsError, JsNativeError, JsResult, JsString, JsValue, NativeFunction,
-    Source,
 };
-use boa_gc::{empty_trace, Finalize, Trace};
+use boa_gc::{Finalize, Trace, empty_trace};
 use core::borrow::Borrow;
 
 /// Converts the given `JsValue` to a `serde_json::Value`.
@@ -241,7 +241,7 @@
         Err(_) => {
             return Err(JsError::from_native(
                 JsNativeError::typ().with_message("invalid salt type"),
-            ))
+            ));
         }
     };
     let initcode = args.get_or_undefined(2).clone();
@@ -385,6 +385,7 @@
 #[cfg(test)]
 mod tests {
     use super::*;
+    use alloc::vec;
 
     #[test]
     fn test_install_bigint() {
@@ -449,7 +450,7 @@
     {
         let array = array.borrow();
         let array = array.as_object().unwrap();
-        let array = JsUint8Array::from_object(array.clone()).unwrap();
+        let array = JsUint8Array::from_object(array).unwrap();
         array.length(&mut Context::default()).unwrap()
     }
 
