--- upstream/tests/codegen/memory/base_cache_keccak.evm
+++ evm2/crates/jit/tests/codegen/memory/base_cache_keccak.evm
@@ -12,8 +12,8 @@
     MSTORE
     STOP

-; CHECK-COUNT-2: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
-; CHECK-NOT: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
-; CHECK: @__revmc_builtin_keccak256
+; CHECK-COUNT-2: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
+; CHECK-NOT: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
+; CHECK: @__evm2_jit_builtin_keccak256
 ; CHECK-NOT: load ptr
-; CHECK-NOT: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
+; CHECK-NOT: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
