--- upstream/tests/codegen/memory/solidity_prologue.evm
+++ evm2/crates/jit/tests/codegen/memory/solidity_prologue.evm
@@ -4,6 +4,6 @@
     STOP

 ; CHECK-NOT: ecx.mem_len
-; CHECK-COUNT-1: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
+; CHECK-COUNT-1: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
 ; CHECK-NOT: ecx.mem_len
-; CHECK-NOT: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
+; CHECK-NOT: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
