--- /home/doni/github/paradigmxyz/revm-inspectors/tests/it/main.rs	2026-05-28 02:57:05.688697684 +0200
+++ /home/doni/github/danipopes/evm2.1/crates/inspectors/tests/it/main.rs	2026-05-15 02:24:57.713240959 +0200
@@ -8,7 +8,7 @@
 
 #[cfg(feature = "std")]
 mod geth;
-#[cfg(feature = "js-tracer")]
+#[cfg(all(feature = "std", feature = "js-tracer"))]
 mod geth_js;
 #[cfg(feature = "std")]
 mod parity;
