--- upstream/tests/codegen/memory/base_cache_after_builtin.evm
+++ evm2/crates/jit/tests/codegen/memory/base_cache_after_builtin.evm
@@ -9,6 +9,6 @@
     MSTORE
     STOP

-; CHECK-COUNT-1: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
+; CHECK-COUNT-1: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
 ; CHECK-COUNT-2: = load ptr, ptr %ecx.mem_base.addr
-; CHECK-NOT: call {{.*}}@__revmc_builtin_mresize{{.*}}; >>
+; CHECK-NOT: call {{.*}}@__evm2_jit_builtin_mresize{{.*}}; >>
